### PR TITLE
Make ubernetes lite e2e tests run stable tests and in parallel.

### LINKED ISSUE #21797: Make ubernetes lite e2e tests run in parallel
... and skip all disruptive, slow, serial, flaky and feature tests.

Right now, all of the above classes of test run by default, due, apparently, to incorrect defaulting?

### PR DESCRIPTION
**Pull request by ghost**
Fixes #21797 

@kubernetes/goog-testing @kubernetes/sig-testing

### CODE REVIEW

**Comment by spxtr** [comment]
LGTM

File: hack/jenkins/job-configs/kubernetes-e2e.yaml [hunk 1]
```diff
@@ -149,12 +149,14 @@
                 export GINKGO_TEST_ARGS="--ginkgo.focus=\[Feature:Example\]"
                 export PROJECT="k8s-jkns-e2e-examples"
         - 'gce-ubernetes-lite':
-            description: 'Run all non-flaky, non-slow tests on GCE, sequentially, and in a multi-zone (Ubernetes-lite) cluster.'
+            description: 'Run all non-flaky, non-slow, non-disruptive, non-flaky, non-feature tests on GCE, in parallel, and in a multi-zone (Ubernetes-lite) cluster.'
```
Comment (spxtr) [inline hack/jenkins/job-configs/kubernetes-e2e.yaml]: You say non-flaky twice here.

**Comment by quinton-hoole** [comment]
@spxtr many thanks for the prompt review.  Fixed the comment typo.  LGTM?

**Comment by spxtr** [comment]
LGTM

**Pull request merged**